src/common/utils/deploy.utils.ts

```diff
@@ -99,4 +102,12 @@ export default class DeployUtils {
     }
     return interfaces_map;
   }
+
+  static async hasClusterDeployment(app: AppService, cluster: Cluster): Promise<boolean> {
+    if (cluster && !cluster.properties.is_shared && !cluster.properties.is_managed) {
+      const cluster_deployment: Deployment = (await app.api.get(`clusters/${cluster.id}/apps`)).data;
+      return Boolean(cluster_deployment) && cluster_deployment.action !== 'delete';
```

Comment:
Why does this need to be cast to a `Boolean`?

Reply:
It doesn't like it when I did just `cluster_deployment`.

Reply:
What's the "it" in this case?

Reply:
I thought VS code didn't like it but I just checked it and it was fine. I'll push up the change.